src/index.js

```diff
@@ -14,7 +14,7 @@ import Pane from './components/pane/pane';
 import ProgressBar from './components/progress-bar/progress-bar';
 import RadioGroup from './components/radio-group';
 import SegmentedControl from './components/segmented-control';
-import SharevilleLogo from './components/shareville-logo';
+import SharevilleLogo from './components/shareville-logo/shareville-logo';
```

Comment:
could you have index.js file, where you re-export stuff, so components' declarations will be consistent?

Reply:
@iamstarkov fixed